src/client/src/apps/MainRoute/widgets/workspace/utils.ts

```diff
@@ -1,12 +1,10 @@
 import { ExternalWindowProps } from './selectors'
-import { TileView } from './workspaceHeader'
 
-export const appendTileViewToUrl: (
+export const createPopOutUrl: (
   externalWindowProps: ExternalWindowProps,
-  tileView: TileView
-) => ExternalWindowProps = (externalWindowProps, tileView) => {
+) => ExternalWindowProps = (externalWindowProps) => {
   const { config } = externalWindowProps
-  const url = `${config.url}?tileView=${tileView}`
+  const url = config.url
```

Comment:
Now that we are removing the query string parameter, is this function necessary at all?

Reply:
You're right, I think it's not needed. Removed it.